hack/generated/pkg/util/armresourceresolver/resolver_test.go

```diff
@@ -26,9 +26,22 @@ import (
 	storage "github.com/Azure/k8s-infra/hack/generated/_apis/microsoft.storage/v20190401"
 )
 
-func NewTestResolver(s *runtime.Scheme) *Resolver {
+func NewTestResolver(s *runtime.Scheme, reconciledGroupVersionsMap map[string]schema.GroupVersion) *Resolver {
 	fakeClient := fake.NewFakeClientWithScheme(s)
-	return NewResolver(kubeclient.NewClient(fakeClient, s))
+
+	return NewResolver(kubeclient.NewClient(fakeClient, s), reconciledGroupVersionsMap)
+}
+
+func MakeGroupToVersionMap(scheme *runtime.Scheme) (map[string]schema.GroupVersion, error) {
+	result := make(map[string]schema.GroupVersion)
+	for gvk := range scheme.AllKnownTypes() {
+		if gvk.Group == "" {
+			continue
+		}
+		result[gvk.Group] = schema.GroupVersion{Group: gvk.Group, Version: gvk.Version}
```

Comment:
Ditto here - do we need to avoid overwrites?

Reply:
Since this is used purely in test it's not a big deal, but I've fixed this as part of the refactor to solve the group-kind issue mentioned above.